DBMS/exe1

```diff
@@ -0,0 +1,13 @@
+Query 1:Find the Pincode with highest average sale price.
+
+
+answer:with temprarytable(pincode,highest_avg) as (select postcode,avg(price) as price from land_registry_price_paid_uk
```

Comment:
Please optimize..try to solve without subquery

Reply:
ok sir

select postcode as pincode from land_registry_price_paid_uk group by pincode order by avg(price) desc  limit 1;